### ISSUE TITLE
proxy状态下如何访问builtin_services

### ISSUE THREAD

**Issue by yanwei1983** [brpc/brpc#331]
当指定了brpc::ServerOptions.http_master_service时，是否还有其他方法能够访问builtin_services

**Comment by jamesge** [comment]
这个需要用户自己去调用

**Comment by yanwei1983** [comment]
在master_service收到请求后，判断是否是internal_port然后，再去访问builtin_services的callmothed吗？

**Comment by jamesge** [comment]
对, `if (!server->options().security_mode() || cntl->local_side().port == server->options().internal_port) { /*allow access to builtin services*/ }
不过一般proxy只需要代理几个url，定义下[restful url](https://github.com/brpc/brpc/blob/master/docs/en/http_service.md#restful-url)就行了，http_master_service在brpc中也就被工具rpc_view用到了。
